spawns.lua

```diff
@@ -128,38 +128,40 @@ NorthGeorgiaFARPTransportSpawns = {
 
 -- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
 
-BladderPos = {}
-BladderPos.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
-BladderPos.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+WeatherPositioning = {}
+WeatherPositioning.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
+WeatherPositioning.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+WeatherPositioning.hClearance		= 305		-- meters [1000']: Default clearance to cloud
 
-function BladderPos.InitOnGroup(planeGroup, vSpeed)
-	local planeGroupName	= planeGroup:getName()
-	vSpeed = vSpeed or BladderPos.vSpeed
-	
+function WeatherPositioning.getCloudFreeAltitude()
 	local clouds = env.mission.weather.clouds
```

Comment:
I think the clouds object should be passed in. This is a pretty kick ass function that can be used based on whatever clouds we want. As long as we pass in something with the base/thickness attributes, we can figure out a "cloud free" altitude for it.

Reply:
In fact, you could probably even use this function to find a de-conflicting altitude for two or more tankers.

Reply:
If not the clouds object, then the base and thickness as two arguments.

Reply:
I'll change it to a more generic sounding deconfliction function (base, top)